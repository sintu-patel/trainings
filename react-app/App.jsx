import React from 'react';
import Container1 from './components/container-1/Container1.jsx'
var defaultData = {
	name: 'TestX'
};
class App extends React.Component {
	render() {
		var data = {
			name: 'Test'
		};
		return (
				<div className="main-app">
					Hello
					<br />
					<Container1 />
				</div>
		);
	}
}
export default App;