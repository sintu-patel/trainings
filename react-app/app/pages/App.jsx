import React from 'react';
import { Link } from 'react-router-dom';
import Button from '../components/button/Button.jsx'

class App extends React.Component {
	render() {
		return (
			<div className="main-app">
				Hello
				<br />
				<Button text='Hello button' />
				<Link to="/test">Test</Link>
			</div>
		);
	}
}
export default App;