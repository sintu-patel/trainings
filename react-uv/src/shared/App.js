import React from 'react';
import PropTypes from 'prop-types';
import Home from './pages/home/Home';
import AboutUs from './pages/about-us/AboutUs';

import {
  Route,
  Switch,
  Link
} from 'react-router-dom';

const App = () => (
  <div className="App">
    <Route path="/" component={ ({ match }) => (
      <div>
        <Switch>
          <Route exact path="/" component={Home} />
          <Route exact path="/test\/:id" component={AboutUs} />
          <Route component={AboutUs}/>
        </Switch>
      </div>
    )}/>
  </div>
)

export default App
