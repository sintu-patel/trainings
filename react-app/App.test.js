import React from 'react';
import {mount} from 'enzyme';
import renderer from 'react-test-renderer';
import App from './App.jsx';

describe('App', () => {
    
    const wrapper = mount(<App />);
    test('App Renders', () => {
        expect(wrapper.find('.main-app').length).toBe(1);
    });
});
