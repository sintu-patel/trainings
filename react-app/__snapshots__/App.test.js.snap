// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot is equal to previous snapshot 1`] = `
<div
  className="main-app"
>
  Hello
  <br />
  <button
    className="button"
    onClick={[Function]}
    type="button"
  >
    Button
  </button>
</div>
`;
